CorsixTH/Lua/hospital.lua

```diff
@@ -257,27 +257,38 @@ function Hospital:Hospital(world, avail_rooms, name)
   end
 end
 
---! Messages regarding numbers cured and killed
+--! Give the user possibly a message about a cured patient.
 function Hospital:msgCured()
-  local msg_chance = math.random(1, 15)
-  if self.num_cured > 1 then
-    if msg_chance == 3 and self.msg_counter > 10 then
+  self.world.ui:playSound("cheer.wav") -- This sound is always heard
+
+  if self.num_cured < 1 then -- First cure is always reported.
+    self.world.ui.adviser:say(_A.information.first_cure)
+
+  elseif self.num_cured > 1 and self.msg_counter > 10 then
+    local msg_chance = math.random(1, 15)
+    if msg_chance == 3 then
       self.world.ui.adviser:say(_A.level_progress.another_patient_cured:format(self.num_cured))
       self.msg_counter = 0
-    elseif msg_chance == 12 and self.msg_counter > 10 then
+    elseif msg_chance == 12 then
       self.world.ui.adviser:say(_A.praise.patients_cured:format(self.num_cured))
       self.msg_counter = 0
     end
   end
 end
---! So the messages don't show too often there will need to be at least 10 days before one can show again.
+
+--! Give the user possibly a message about a dead patient.
 function Hospital:msgKilled()
-  local msg_chance = math.random(1, 10)
-  if self.num_deaths > 1 then
-    if msg_chance < 4 and self.msg_counter > 10 then
+  self.world.ui:playSound("boo.wav") -- this sound is always heard
+
+  if self.num_deaths < 1 then -- First death is always reported.
+    self.world.ui.adviser:say(_A.information.first_death)
+
+  else if self.num_deaths > 1 and self.msg_counter > 10 then
```

Comment:
Needs to be ``elseif`` here, probably cause of the travis error.

Reply:
Thanks!